rumqttc/src/lib.rs

```diff
@@ -315,7 +319,7 @@ impl Transport {
 }
 
 /// TLS configuration method
-#[derive(Clone, Debug)]
+#[derive(Clone)]
```

Comment:
derive of Debug is removed because TlsConnectorBuilder has no Debug trait, not sure if there is better idea for it.

Reply:
Close the discuss. Use TlsConnector which has debug trait.